package.json

```diff
@@ -25,5 +25,8 @@
     "stylelint": "^13.12.0",
```

Comment:
Shouldn't we upgrade this to 14?

Reply:
Yes, I ran `npm install --save-dev stylelint-config-standard-scss` this command.
Should I change it to 14.0.0 manually?

Reply:
Yeah I think we should upgrade to `14...` manually then re-run `npm i`